shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -43,6 +45,19 @@ - (BOOL)addSkipBackupAttributeToItemAtPath:(NSString *) filePathString
   return success;
 }
 
+- (void) createBkgReadableDir:(NSString*) path
```

Comment:
This sets attribute on file no directory.. also you can be verbose in objective-c... so maybe `enableBackgroundOpenableFile`